app/css/interceptor.css

```diff
@@ -707,3 +707,7 @@ svg.spinner > circle {
 	padding-bottom: 10px;
 	padding-top: 10px;
 }
+
+.blur {
+	filter: blur(3px);
```

Comment:
Can we make this a bit less blurry so you can still mostly make out the values behind the blur?  In most cases, when re-simulating the results won't change and it will be annoying if you are trying to read it and every 12 seconds it blurs and you have to pause for a second to read.

Reply:
one option is to make it blur only on clear situations when we know the simulation results are wrong. Eg, when you change account, change chain, add new transaction

Reply:
I would be far more in favor of that.

Reply:
but not like that either? My problem is that the simulation takes a bit time, and also the more stuff we add to it, the longer it takes, and we need to somehow display to users that it waits for simulation

Reply:
I am OK with a blur if we are reasonably confident that the results are going to change.  The thing I'm minor 👎 on is having it blur every 12 seconds when we re-simulate just because a new block arrived.